django/template/loaders/eggs.py

```diff
@@ -20,19 +31,42 @@ def __init__(self, engine):
             raise RuntimeError("Setuptools must be installed to use the egg loader")
         super(Loader, self).__init__(engine)
 
+    def get_contents(self, origin):
+        try:
+            source = resource_string(origin.app_name, origin.pkg_name)
+        except:
```

Comment:
Yeah, the bare except isn't optimal. That's behavior I carried over from the existing egg loader.

I didn't change it since I don't know what exceptions are raised with real eggs. The test suite isn't helpful here since we use a mock egg provider. Do you want me to look into that? Or can we make it a separate ticket for now?